sdk/ml/azure-ai-ml/azure/ai/ml/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
This file shouldn't be inside the azure/ai/ml package. Could we please remove it?

Reply:
Removed.